h2o-core/src/main/java/water/H2O.java

```diff
@@ -2171,7 +2171,11 @@ public static void setFlatfile(Set<H2ONode> nodes) {
   public static H2ONode reportClient(H2ONode client) {
     H2ONode oldClient = CLIENTS_MAP.put(client.getIpPortString(), client);
     if (oldClient == null) {
-      Log.info("New client discovered at " + client);
+      if (client._heartbeat._watchdog_client) {
+        Log.info("New watchdog client discovered at " + client);
```

Comment:
It would be more elegant without the if, either client.toString() or client.toDebugString() could include the information

Reply:
Good point